udfs/community/ts_gen_keyed_timestamps.sql

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+* Generate an array of key-timestamp structs with the specified min, max and interval timeseries
+* Expected Usage:
+* SELECT *
+* FROM UNNEST(bqutil.fn.ts_gen_keyed_timestamp(['abc'], 900, '2020-01-01', '2020-01-02') a
+* LEFT JOIN dataset.table ON a.series_key = a.key AND a.tumble_val = b.timestamp
+*/
+CREATE OR REPLACE FUNCTION fn.ts_gen_keyed_timestamps(
```

Comment:
Structure above comment.

Similar to:
https://github.com/GoogleCloudPlatform/bigquery-utils/blob/master/udfs/community/csv_to_struct.sql